Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -2,13 +2,10 @@
   "ValidatorsAssemblyName": "Security.dll",
   "Definitions": [
     {
-      "SharedStrings": {
-        "$BinaryFiles": "(?i)\\.(bmp|dll|exe|gif|jpg|jpeg|lock|png|psd|tar\\.gz|tif?|ttf|xcf|zip)$",
-        "$SourceFiles": "(?i)\\.(azure|bat|c|cmd|config|cpp|cs|cscfg|definitions|dtsx|h|hxx|hpp|ini|java|jsx?|json|keys|kt|loadtest|m|md|php|properties|ps1|psm1|pubxml|py|resx|sample|sql|ste|swift|test|tsx?|txt|waz|xml)$"
-      },
       "Id": "SEC101",
       "Name": "DoNotExposePlaintextSecrets",
       "Level": "Error",
+      "FileNameDenyRegex": "$BinaryFiles",
```

Comment:
>FileNameDenyRegex [](start = 7, length = 17)

Oops, our security analysis was running against all files due to an incomplete earlier change.